src/SignalR/perf/benchmarkapps/BenchmarkServer/Startup.cs

```diff
@@ -23,6 +23,7 @@ public void ConfigureServices(IServiceCollection services)
             {
                 o.EnableDetailedErrors = true;
             })
+            // TODO: Json vs NewtonsoftJson option
```

Comment:
Are you going to do this before merging?

Reply:
Nope